src/app/services/authentication.service.ts

```diff
@@ -1,17 +1,32 @@
 import { Injectable } from '@angular/core';
+import {HttpClient,HttpHeaders} from '@angular/common/http';
+import {environment} from '../../environments/environment';
+
+class User {
+  username: string;
+  password: string;
+}
 
 @Injectable({
   providedIn: 'root'
 })
 export class AuthenticationService {
 
-  constructor() { }
+  readonly apiUrl = environment.apiUrl;
+
+  constructor(private httpClient: HttpClient) {}
 
-  private getUserInfo(){
+  public loginUser(user: User){
     //authenticates user while login attempt
+    const options = {
+      headers: new HttpHeaders({
+        "Content-Type":"application/json",
+      })
+    }
+    return this.httpClient.post(`${this.apiUrl}/api-token-auth/`,JSON.stringify(user),options)
```

Comment:
I might be wrong, but I think our backend does not accept json format. Did you test this?

Reply:
I had a problem creating users with your backend, the picture in your pull request does not indicate the user's password, so I was unable to create users and test it. Was just in the process of asking this on your pr

Reply:
Could you help me on this?

Reply:
@Khukhuna backend does accept application/json!

Reply:
Sorry for late reply, I'm in school atm, Try:
**Username:** test
**Password:**  jbossoutreach

You can send data as:
application/x-www-form-urlencoded

Reply:
That doesn't seem to work, i get 
```js
{
    "non_field_errors": [
        "Unable to log in with provided credentials."
    ]
}
```

when using either `application/json` or `application/x-www-form-urlencoded`.

Reply:
I tried
```js
{
	"user":{
		"username":"abcde",
		"first_name":"jun",
		"last_name":"ron",
		"email":"fake@fake.com",
		"password":"abcde"
	},
	"name":"abcde",
	"password":"abcde"
}
```
for creating users, it seems to have no error, but login still doesn't work

Reply:
Actually creating users is still in progress! There are a few tweaks that are left but I have given login details to @nikitadhiman for student and organization! Until we fix the registration system you can use those credentials